src/BenchmarksServer/Startup.cs

```diff
@@ -3792,14 +3791,19 @@ private static void StartDotNetTrace(int processId, ServerJob job)
             }
         }
 
-        private static async Task UseMonoRuntimeAsync(string runtimeVersion, string outputFolder)
+        private static async Task UseMonoRuntimeAsync(string runtimeVersion, string outputFolder, string mode)
         {
-            var monoRuntimeUrl = String.Format(_runtimeMonoPackageUrl, runtimeVersion);
-
             try
             {
-
-                var packageName = "Microsoft.NETCore.App.Runtime.Mono.linux-x64".ToLowerInvariant();
+                var packageName = "";
+                switch (mode) {
+                    case "jit":
+                        packageName = "Microsoft.NETCore.App.Runtime.Mono.linux-x64".ToLowerInvariant();
+                        break;
+                    case "llvm-jit":
+                        packageName = "Microsoft.NETCore.App.Runtime.Mono.LLVM.AOT.linux-x64".ToLowerInvariant();
```

Comment:
Is this right or should we use the LLVM.JIT package ? Seems confusing

Reply:
We don't have the LLVM.JIT package. For some reason, they never got generated. The literal only difference between the AOT LLVM and JIT LLVM packages are the AOT packages also contain llc/opt. So it is totally fine to use the AOT LLVM package for JIT LLVM.